meta-intel-edison-distro/conf/distro/poky-edison.conf

```diff
@@ -13,6 +13,7 @@ PREFERRED_VERSION_nodejs = "6.%"
 PREFERRED_VERSION_nodejs-native = "6.%"
 BINDINGS_pn-mraa="python nodejs"
 BINDINGS_pn-upm="python nodejs"
+PREFERRED_VERSION_cross-localedef-native = "2.26.%"
```

Comment:
This one is not necessary and actually produces a warning, as there's no package with version 2.26.something - only 2.26. however as mentioned in my other comment, 2.26 is anyway the newest one and with our layer config, it will be picked automatically.